docs/diagrams/DataTransferClasses.puml

```diff
@@ -0,0 +1,68 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+class DataBundle
+class "InstructorAttributes" as IA1
+class "StudentAttributes" as SA1
+class CourseAttributes
+class AccountAttributes
+class "*Attributes" as Attributes
+
+class "*SearchResultBundle" as SearchResultBundle
```

Comment:
It seems like this is displayed as a dot: i.e.

- SearchResultBundle

Rather than
`*SearchResultBundle`.
It isn't clear that the former is a pointer, while the latter is, can we change it so that it shows the latter? (Similar to your other diagrams using a *)

Reply:
I'm not sure why plantuml recognises asterisks as bullet points for class names but I added an additional space and it seemed to solve the issue!